### PR TITLE
Added username and password from environmental variables

### PR DESCRIPTION
**Pull request by hwd15508**
see title

### CODE REVIEW

File: nidata/multimodal/hcp/datasets.py [hunk 1]
```diff
@@ -14,6 +14,8 @@ def fetch(self, files, force=False, resume=True, check=False, verbose=1):
         if self.jsession_id is None:
             # Log in to the website.
             import requests
+            self.username = os.environ.get("NIDATA_USERNAME")
+            self.passwd = os.environ.get("NIDATA_PASSWD")
```
Comment (bcipolli) [inline nidata/multimodal/hcp/datasets.py]: `self.username` is set inside `__init__`, so I think this should happen there. Also, if `username` was specified in `__init__`, it gets overwritten here. I'd rather that these environment variables are only used *if `username` and/or `password` are not passed directly in `**init**.

**Comment by bcipolli** [comment]
@hwd15508 please take a look and update. Thanks!

**Comment by hwd15508** [comment]
@bcipolli made the changes

**Comment by bcipolli** [comment]
Looks good; in it goes!

**Pull request merged**